tools/configwrappers/config.go

```diff
@@ -0,0 +1,163 @@
+package configwrappers
+
+import (
+	"math/big"
+	"sort"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/ccip-owner-contracts/tools/errors"
+	"github.com/smartcontractkit/ccip-owner-contracts/tools/gethwrappers"
+)
+
+// Config is a struct that holds all the configuration for the owner contracts
+type Config struct {
+	Quorum uint8 `json:"quorum"`
+
+	// TODO: how should this change as we expand to other non-EVM chains?
+	Signers      []common.Address `json:"signers"`
+	GroupSigners []Config         `json:"groupSigners"`
+}
+
+func NewConfig(quorum uint8, signers []common.Address, groupSigners []Config) *Config {
+	return &Config{
+		Quorum:       quorum,
+		Signers:      signers,
+		GroupSigners: groupSigners,
+	}
+}
+
+func NewConfigFromRaw(rawConfig gethwrappers.ManyChainMultiSigConfig) *Config {
+	groupToSigners := make([][]common.Address, len(rawConfig.GroupQuorums))
+	for _, signer := range rawConfig.Signers {
+		groupToSigners[signer.Group] = append(groupToSigners[signer.Group], signer.Addr)
+	}
+
+	groups := make([]Config, len(rawConfig.GroupQuorums))
+	for i, quorum := range rawConfig.GroupQuorums {
+		signers := groupToSigners[i]
```

Comment:
is `group` just an index? since above we set `groupToSigners[signer.Group]` but now we are just doing `groupToSigners[i]`